### DISCUSSION TITLE
간편링크

### DISCUSSION THREAD

**Discussion by jun5640** [naverz/zepeto-studio-kor#224] [5. Q&A]
요즘 제페토 맵에 접속하면 간편 초대 링크를 제공하더라고요.
해당 초대 링크를 월드에서 제작한 맵에서 제공하고 싶은데 방법이 있을까요?

**Comment by moondory77-naverz** [comment]
안녕하세요. 
ZEPETO.World 1.1.2 패키지 기준, 월드에 대한 초대 링크 기능을 제공드리고 있습니다.
[1.1.2 ReleaseNote](https://studio.zepeto.me/guides/world-112)

(기본적으로 World 초대가능으로 설정되어 있으며, 옵션을 통해 조정 가능 합니다.)

초대 기능이 활성화 되어있을 시, 아래 스크린 샷과 같이 초대링크를 제공하며
이를 통해 월드 접속이 가능합니다.

![친구초대](https://user-images.githubusercontent.com/93751484/150055976-0736fbb6-ade7-4c71-8a1d-6effe8eebfe9.png)

도움이 될 수 있다면 좋겠습니다.
감사합니다.

**Reply by jun5640** [reply]
답변 감사드립니다.
혹시 해당 링크 말고도 인터넷에 링크를 입력하면 제페토가 설치 안된 기기면 제페토 설치 화면이 설치가 된 상태라면 해당 월드에 자동 방생성 및 접속가능하게 하는 링크를 할 수 있는 방법이 있을까요?
